view-src/src/theme.js

```diff
@@ -0,0 +1,28 @@
+import createMuiTheme from '@material-ui/core/styles/createMuiTheme';
+
+const theme = createMuiTheme({
```

Comment:
write in different CSS file and import that here

Reply:
@laveesh i think css in jss would be more conformable from there we would be able to take the full advantage of the material-ui 

while with the css it is the alpa version we can shift after wards if version is stable https://material-ui.com/css-in-js/basics/

Reply:
> @laveesh is this PR reviewed from your end?

Yes it is now it can merged and I am satisfied with his answers